packages/opentelemetry-exporter-collector/test/browser/CollectorTraceExporter.test.ts

```diff
@@ -98,7 +98,7 @@ describe('CollectorTraceExporter - web', () => {
 
           assert.strictEqual(stubOpen.callCount, 0);
 
-          assert.strictEqual(header, collectorExporterConfig.contentTypeBeacon);
+          assert.strictEqual(header, collectorExporterConfig.beaconBlobPropertyBag?.type);
```

Comment:
small thing, header in assertion can be renamed to a meaningful word , respecting beacon context ...

Reply:
I renamed it `blobType`